src/pages/contact.tsx

```diff
@@ -92,36 +92,17 @@ const Contact: React.FC = () => {
       if (typeof window.heap !== 'undefined') {
         window.heap.track('contact', formData)
       }
-      if (typeof window.gtag !== 'undefined') {
-        window.gtag('event', 'conversion', {
-          send_to: 'AW-877332159/jdvuCLLdpdQBEL-NrKID',
-        })
-        window.gtag('event', 'conversion', {
-          send_to: 'AW-616173950/7ehtCOKFvNYBEP6i6KUC',
+      if (typeof window.analytics !== 'undefined') {
```

Comment:
Could this be falsey tested instead of just undefined?

Reply:
Yeah I can change this